internal/auth/oidc/auth_method.go

```diff
@@ -77,7 +77,7 @@ func NewAuthMethod(scopeId string, clientId string, clientSecret ClientSecret, o
 			ScopeId:          scopeId,
 			Name:             opts.withName,
 			Description:      opts.withDescription,
-			OperationalState: string(InactiveState),
+			OperationalState: string(opts.withOperationalState),
```

Comment:
I had intentionally not allowed this... we allow "incomplete" authmethods to be created and these incomplete methods should stay inactive until they are complete.   If we want to support setting the state to anything but "inactive" then we need to ensure it isCompete() and if not raise an error.

Reply:
Fixed.